src/EditorFeatures/CSharpTest/Classification/SyntacticClassifierTests.cs

```diff
@@ -460,35 +468,49 @@ public async Task LinqWhere2()
                 String(@"""bar"""));
         }
 
-        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/44423"), Trait(Traits.Feature, Traits.Features.Classification)]
+        [Theory, Trait(Traits.Feature, Traits.Features.Classification)]
         [WorkItem(44423, "https://github.com/dotnet/roslyn/issues/44423")]
-        public async Task VarContextualKeywordAtNamespaceLevel()
+        [CombinatorialData]
+        public async Task VarContextualKeywordAtNamespaceLevel(bool script, bool outOfProcess)
         {
             var code = @"var goo = 2;";
+
+            var parseOptions = script ? Options.Script : null;
+
             await TestAsync(code,
                 code,
-                Identifier("var"),
-                Field("goo"),
+                parseOptions,
+                outOfProcess,
+                script ? Identifier("var") : Keyword("var"),
+                script ? Field("goo") : Local("goo"),
                 Operators.Equals,
                 Number("2"),
                 Punctuation.Semicolon);
         }
```

Comment:
i don't love this.  but i don't hate it enough to protest.

Reply:
I mostly like that it shows how the same input can be interpreted in specific different ways based on the content type.